libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -226,14 +266,80 @@ void AP_GPS::init(DataFlash_Class *dataflash, const AP_SerialManager& serial_man
     _port[0] = serial_manager.find_serial(AP_SerialManager::SerialProtocol_GPS, 0);
     _port[1] = serial_manager.find_serial(AP_SerialManager::SerialProtocol_GPS, 1);
     _last_instance_swap_ms = 0;
+
+    // Initialise class variables used to do GPS blending
+    _omega_lpf = 1.0f / constrain_float(_blend_tc, 5.0f, 30.0f);
+
+    // sanity check update rate
+    for (uint8_t i=0; i<GPS_MAX_RECEIVERS; i++) {
```

Comment:
yes, if the user sets it incorrectly, I imagine that's how we will hear from them.  If I go ahead with removing the code, we likely won't hear from them at all and the user will be none the wiser that what they've requested isn't being done.  It's a slight "peep" of a warning on a mistake that probably isn't vehicle threatening in any case.